views/pulsar/v2/helpers/form.html.twig

```diff
@@ -54,15 +54,31 @@ items         | array  | An array of items to put in the dropdown list (usually
     {% import _self as form %}
     {% import '@pulsar/pulsar/v2/helpers/elem.html.twig' as elem %}
 
-    {{
-        form.group({
-            'parent': options
-                |only('bare class error guidance guidance-container help id label required show-label type')
-                |merge({'class': options.class|default ~ ' form__button-group'}),
-            'controls': {'class': 'btn__group'},
-            'inputs': options.items
-        })
-    }}
+    {% if options.help is defined and options.help is not empty %}
+        {% set help_id = 'guid-' ~ random() %}
+        {% set options = options|merge({ 'help_id': help_id }) %}
+
+        {{
+            form.group({
+                'parent': options
+                    |only('bare class error guidance guidance-container help help_id id label required show-label type')
+                    |merge({'class': options.class|default ~ ' form__button-group', 'aria-describedby': help_id}),
+                'controls': {'class': 'btn__group'},
+                'inputs': options.items
+            })
+        }}
+    {% else %}
```

Comment:
Can we avoid the duplication here? 

1. set a null help_id at the top of the helper which can be overridden in the loop. e.g. https://twigfiddle.com/aiz1nw
2. move the setting of `aria-describedby` so you don't need a separate form.group instance

```
{% set help_id = null %}
{% if options.help is defined and options.help is not empty %}
    {% set help_id = 'guid-' ~ random() %}
    {% 
        set options = options|merge({ 
            'help_id': help_id,
            'aria-describedBy': 'help_id'
        }) 
    %}
{% endif %}

{{ form.group(all the other stuff) }}
```